.woodpecker.yml

```diff
@@ -27,6 +27,28 @@ pipeline:
     commands:
       - prettier -c . '!**/volumes' '!**/dist' '!target' '!**/translations'
 
+  restore-cache:
+    image: meltwater/drone-cache:v1
```

Comment:
Did you check out this woodpecker plugin? https://woodpecker-ci.org/plugins/S3 Plugin

Using a drone one scares me a bit.

Reply:
@dessalines I've just had a look & it appears that the S3 plugin for woodpecker is upload only & doesn't appear to be oriented around caching, rather it's more directed towards uploading/publishing artifacts to S3